scripts/python/lib/validate_config_logic.py

```diff
@@ -301,6 +338,7 @@ def validate_config_logic(self):
         self._validate_dhcp_lease_time()
         self._validate_labels()
         self._validata_software_bootstrap()
+        self._validata_os_profiles()
```

Comment:
is _validata (instead of _validate) a typo or intentional?

Reply:
Typo! I copy/paste/modified the typo from the function defined above it: "_validata_software_bootstrap()" :) Good catch.